Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/LocationUpdateService.java

```diff
@@ -810,6 +813,16 @@ private void showAlertNotification()
                 .setPriority(NotificationCompat.PRIORITY_MAX);
         mBuilder.setContentIntent(pendingIntent);
         NotificationManagerCompat notificationManager = NotificationManagerCompat.from(context);
+        if (ActivityCompat.checkSelfPermission(this, Manifest.permission.POST_NOTIFICATIONS) != PackageManager.PERMISSION_GRANTED) {
```

Comment:
@Anuragini-2506 why is this required here?

Reply:
This was added to solve the GPS issue